theme/static/js/hs_file_browser.js

```diff
@@ -63,7 +63,7 @@ function getFileTemplateInstance(fileName, fileType, aggregation_name, logical_t
     else {
         var title = '' + fileName + "&#13;Type: " + fileType + "&#13;Size: " + formatBytes(parseInt(fileSize));
     }
-    return "<li data-pk='" + pk + "' data-url='" + url + "' data-ref-url='" + ref_url + "' data-logical-file-id='" + logical_file_id + "' class='fb-file draggable' title='" + title + "'>" +
+    return "<li data-pk='" + pk + "' data-url='" + url + "' data-ref-url='" + ref_url + "' data-logical-file-id='" + logical_file_id + "' class='fb-file draggable' title='" + title + " 'single-file=" + single_file + ">" +
```

Comment:
@sblack-usu I believe you missed an ending ```"``` to pair with beginning ```"``` for ```title```

Reply:
hmm, you're right.  This was working but I've updated it to follow your comment.